_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,275 @@
+
+### Container scanning
+
+Snyk can scan your container images for vulnerabilities it can do this from docker images, Amazon ECR, Docker hub, ...
+You can set this up in your Kubernetes cluster, but currently Fargate is not supported.
+
+If you want to automate the updating of your images you can do this by using container scanning on your git repository,
+using this method Snyk can automatically create pull requests for you that you can test and then merge if satisfied.
+This reduces your effort to stay safe and up to date.
+
+### Infrastructure as code scanning
+
+Snyk can scan your infrastructure as code against the CIS AWS Foundations Benchmark or you can write custom policies.
+To scan you IaC you simply have to add your git repository that contains your IaC
+and the Snyk platform will start scanning if for you.
+You can also use the Snyk CLI to scan your IaC if you want to make it part of your CI/CD pipelines.
+By using it this way you can make this a requirement before pull requests are merged that you pass the CLI tool's scan
+or even block deployment to environments.
+
+### Custom policies
+
+Snyk allows you to write custom policies in Rego, but only for IaC scanning and platform policies.
+
+OPA is easy to use because applications can easily delegate police validation to OPA if needed.
+Snyk IaC leverages OPA to do it's polic scanning according to one of their blogposts.
+You get a preset of policies out o the box from Snyk and you can add your own custom policies writen in Rego.
+
+## The Container Detective: Trivy
+
+[Trivy](https://trivy.dev/){:target="_blank" rel="noopener noreferrer"} is an open-source cli tool provided by Aqua
+Security.
+
+<img src="{{ '/img/2023-02-16-secure-cloud-foundation/Trivy.png' | prepend: site.baseurl }}" alt="Gandalf, You shall not pass" class="image fit" style="margin:0px auto; max-width:100%; height:60%">
+
+### Container scanning
+
+Trivy can scan container images against well known vulnerabilities.
+On the [tool's homepage](https://trivy.dev/), you can enter public available docker hub images to test it out.
+Trivy will scan files inside container images and container image metadata.
+
+Trivy scans the files inside container images for:
+
+* Vulnerabilities
+* Misconfigurations
+* Secrets
+* Licenses
+
+The image metadata will be scanned for:
+
+* Misconfigurations
+* Secrets
+
+### Dependency scanning
+
+Trivy can scan your dependencies for well known vulnerabilities.
+It has a mode that automatically discovers, declarations files for various package managers
+This Dependency scanning is very powerful it scans the file system for typical files used to declare dependencies,
+like a `pom.xml`, but can also scan into jar and war files.
+If you thought that's nice well it can also scan your linux systems package managers installed packages, apt and apk are
+supported of the box for alpine and ubuntu based images.
```

Comment:
Try to rephrase this. The grammar is weird, I don't really like the "if you thought that's nice well...", especially without any quotations.

Dependency scanning <-- unnecessary capital letter 'D'